app/src/main/java/net/squanchy/schedule/domain/view/Event.kt

```diff
@@ -0,0 +1,83 @@
+package net.squanchy.schedule.domain.view
+
+import net.squanchy.eventdetails.domain.view.ExperienceLevel
+import net.squanchy.speaker.domain.view.Speaker
+import net.squanchy.support.lang.Optional
+import org.joda.time.DateTimeZone
+import org.joda.time.LocalDateTime
+
+data class Event(
+        val id: String,
+        val numericId: Long,
+        val startTime: LocalDateTime,
+        val endTime: LocalDateTime,
+        val title: String,
+        val place: Optional<Place>,
+        val track: Optional<Track>,
+        val speakers: List<Speaker>,
+        val experienceLevel: Optional<ExperienceLevel>,
+        val dayId: String,
+        val type: Type,
+        val favorited: Boolean,
+        val description: Optional<String>,
+        val timeZone: DateTimeZone
+) {
+
+    fun speakersNames(): String {
```

Comment:
Do we still actually use this? I suppose this should be moved to the UI layer :P